src/lib/ndpi_main.c

```diff
@@ -2060,6 +2060,10 @@ static void ndpi_init_protocol_defaults(struct ndpi_detection_module_struct *ndp
                           "BACnet", NDPI_PROTOCOL_CATEGORY_IOT_SCADA,
                           ndpi_build_default_ports(ports_a, 0, 0, 0, 0, 0) /* TCP */,
                           ndpi_build_default_ports(ports_b, 47808, 0, 0, 0, 0) /* UDP */);
+  ndpi_set_proto_defaults(ndpi_str, 1 /* cleartext */, 0 /* nw proto */, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_HOTS,
+        "Heroes of the Storm", NDPI_PROTOCOL_CATEGORY_GAME,
```

Comment:
No spaces in the name protocol; up to you if switch to "Heroes_of_the_Storm", "HeroesOfTheStorm",...

Reply:
Replaced spaces with underscores.